### PR TITLE
Governance

### PR DESCRIPTION
**Pull request by mike-burns**
Document the contract between maintainer, committer, and contributor.

This documents how it currently works, not how we wish it to work. Start
there so that we can consciously improve.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,3 +76,46 @@ If you intend to write a new test:
...
+
+You are expected to adhere to our [code of conduct].
+
+Committer
+---------
+
+As a committer, you can merge at any time. We encourage you to have the code
+reviewed by `@thoughtbot/shell`.
+
+You are expected to adhere to our [code of conduct].
```
Comment (seanpdoyle) [inline CONTRIBUTING.md]: `[code of conduct]` is mentioned twice, but I don't see a URL declaration anywhere.

Comment (seanpdoyle) [inline CONTRIBUTING.md]: Nevermind, `[code of conduct]` is mentioned earlier in the file.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -76,3 +76,46 @@ If you intend to write a new test:
...
+description of how to write your patch while closing the request, though we
+cannot guarantee this.
+
+You are expected to adhere to our [code of conduct].
+
+Committer
+---------
+
+As a committer, you can merge at any time. We encourage you to have the code
+reviewed by `@thoughtbot/shell`.
```
Comment (gfontenot) [inline CONTRIBUTING.md]: I believe only members of the thoughtbot team will be able to use this mention. Does that matter?

Comment (mike-burns) [inline CONTRIBUTING.md]: You are right, according to the docs. Hm.

I suppose it does not matter in practice: I am a member of `/shell`, and I review PRs on this repo. I'll spell that out instead.

Comment (teoljungberg) [inline CONTRIBUTING.md]: I'll happily review PRs too. Eventhough I'm not a part of the group.

**Comment by mike-burns** [comment]
@derekprior 

> Is it worth specifying how documentation patches are handled as contributions?

Can you expand on that a bit? I have this in there already:

> Bug reports, either as issues or as pull requests, get our priority. We consider documentation bugs to be as important as code bugs.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -83,41 +83,66 @@ Governance
...
+
+All maintainers must adhere to our [code of conduct].
+
 Promotion
 ---------
 
 All current thoughtbot employees are committers to rcm.
 
-Contributors with two merged patches are weclome to request committer access.
+Contributors with two merged patches are weclome to request committer access by
```
Comment (seanpdoyle) [inline CONTRIBUTING.md]: %s/weclome/welcome/

**Pull request merged**